sdk/core/core/src/az_span.c

```diff
@@ -129,7 +129,9 @@ AZ_NODISCARD az_result az_span_to_uint32(az_span span, uint32_t* out_number)
   return AZ_OK;
 }
 
-AZ_NODISCARD az_result az_span_find(az_span source, az_span target, az_span* out_span)
+#define AZ_SPAN_FIND_TARGET_NOT_FOUND -1
```

Comment:
We always makes these enums, not defines.
But, do we want to make this public? If so, then the docs should use the symbol; not -1.
If not, then I'd just define this as a const int inside the az_span_find function.

Reply:
Let's keep it simple then. I'll define a const inside az_span_find then.

Reply:
Done.